### PR TITLE
DDP-6040, DDP-6042, DDP-6045 rarex edits

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

Doing 3 tickets in one PR. See comments for details.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.

## FUD Score

- [x] :relaxed: All good, business as usual!

## How do we demo these changes?

Need to redeploy study.

## Testing

- [x] I have written zero automated tests but have poked around locally to verify proper functionality

## Release

Need to redeploy study.

### CODE REVIEW

File: study-builder/studies/rarex/i18n/en.json [hunk 1]
```diff
@@ -12,20 +12,20 @@
...
-  "prequal_diagnosis_vici": "Vici syndrome",
-  "prequal_diagnosis_dada2": "DADA2 (deficiency of the enzyme ADA2 (Adenosine Deaminase 2)",
-  "prequal_diagnosis_smard": "SMARD (Spinal Muscular Atrophy with Respiratory Distress)",
-  "prequal_diagnosis_shwachman_diamond": "Shwachman Diamond Syndrome",
-  "prequal_diagnosis_other": "Other",
+  "diagnosis_4h_leukodystrophy": "4H Leukodystrophy (hypomyelination, hypogonadotropic hypogonadism and hypodontia)",
+  "diagnosis_ahc": "AHC - Alternating Hemiplegia of Childhood",
+  "diagnosis_bpan_wdr45": "BPAN/WDR45 Beta-Propeller Protein-Associated Neurodegeneration",
+  "diagnosis_cacna1achamp1": "CACNA1ACHAMP1 - Chromosome Alignment- Maintaining Phosphoprotein",
+  "diagnosis_chd2": "CHD2 - CHD2 myoclonic encephalopathy",
```
Comment (yufengwng) [inline study-builder/studies/rarex/i18n/en.json:19]: Don't know why it's call prequal here. The disease list is not used in the prequal, so I renamed stuff. The `CHD2` typo is fixed here for DDP-6040.

Screen Shot 2021-05-11 at 1 32 25 PM

File: study-builder/studies/rarex/data-sharing.conf [hunk 2]
```diff
@@ -765,27 +765,47 @@
           "blockType": "CONTENT",
           "shownExpr": null
         },
+        {
+          "titleTemplate": null,
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """
+                  <h4>$data_sharing_biospecimen_interested</h4>
+                  <p><span style="font-weight:normal">$data_sharing_contacted_when_available</span></p>
```
Comment (yufengwng) [inline study-builder/studies/rarex/data-sharing.conf:774]: DDP-6042 "biospecimen(s)" question.

Screen Shot 2021-05-11 at 1 31 28 PM

File: study-builder/studies/rarex/data-sharing.conf [hunk 3]
```diff
@@ -1045,27 +1063,47 @@
           "blockType": "CONTENT",
           "shownExpr": null
         },
+        {
+          "titleTemplate": null,
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """
+                  <h4>$data_sharing_other_data_sources_interested</h4>
+                  <p><span style="font-weight:normal">$data_sharing_contacted_when_available</span></p>
```
Comment (yufengwng) [inline study-builder/studies/rarex/data-sharing.conf:1072]: DDP-6042 "other possible data sources" question.

Screen Shot 2021-05-11 at 1 31 49 PM

File: study-builder/studies/rarex/i18n/en.json [hunk 4]
```diff
@@ -639,39 +640,36 @@
...
+  "data_sharing_other_data_sources_interested": "We know that you may have participated in other studies/data collection. Are you interested in having the data from those studies connected to your data in RARE-X? *",
   "data_sharing_other_data_sources_interested_yes": "Yes",
   "data_sharing_other_data_sources_interested_no": "No",
   "data_sharing_other_data_sources_list_of_studies": "If you remember, please list the name of the study/studies, the researcher(s) or the location(s) of the study (studies).",
   "data_sharing_other_data_sources_list_of_studies_req_hint": "This field should not be empty",
   "data_sharing_other_data_sources_list_of_studies_max_hint": "Text maximum length is 500",
   "data_sharing_known_data_sources_intro": "Patient Community Connections",
-  "data_sharing_pags_want_to_share": "Do you want to share your contact information with any patient advocacy groups that support your diagnosis(s)?",
+  "data_sharing_pags_want_to_share": "Do you want to share your contact information with patient advocacy groups that support your diagnosis(s)?",
```
Comment (yufengwng) [inline study-builder/studies/rarex/i18n/en.json:669]: DDP-6042 "patient community connections" question.

Screen Shot 2021-05-11 at 1 32 00 PM

File: study-builder/studies/rarex/general-information.conf [hunk 5]
```diff
@@ -2618,6 +2614,24 @@
                   ]
                 },
                 "validations": [
+                  {
+                    "ruleType": "REQUIRED",
```
Comment (yufengwng) [inline study-builder/studies/rarex/general-information.conf:2618]: DDP-6045 race question.

Screen Shot 2021-05-11 at 1 55 57 PM

Screen Shot 2021-05-11 at 1 56 05 PM

Comment (MocanaAtBroad) [reply study-builder/studies/rarex/general-information.conf:2618]: Do we need to tweak styling in Angular for indentation?

**Review by MocanaAtBroad — DISMISSED** [review]

**Review by YuriyPavlovBI — DISMISSED** [review]

File: study-builder/studies/rarex/general-information.conf (hunk 5 above)
Comment (yufengwng) [reply study-builder/studies/rarex/general-information.conf:2618]: My screenshot-ing skills was terrible so it looks off, but indentation is fine.

**Comment by yufengwng** [comment]
Per latest comments in DDP-6045, we're not making "race" question required yet, so removed the `REQUIRED` rule.

**Review by YuriyPavlovBI — DISMISSED** [review]

**Pull request merged**